speechbrain/utils/checkpoints.py

```diff
@@ -94,10 +131,13 @@ def torch_recovery(obj, path, end_of_epoch):
     """
     del end_of_epoch  # Unused
     device = "cpu"
+
+    state_dict = torch.load(path, map_location=device)
+    state_dict = map_old_state_dict_weights(state_dict, KEYS_MAPPING)
```

Comment:
Shouldn't we have a more generic `state_dict` fixup function that would call into `map_old_state_dict_weights`, even if the general function only calls that one for now?

Reply:
I don't understand your question sorry haha

Reply:
Having a function where we could add other `state_dict` fixes in the future.

Reply:
so you mean something like:
```python
def foo(state_dict):
	state_dict(state_dict)
```

so that someone can do : 
```python
def foo(state_dict):
	bar(state_dict)
	state_dict(state_dict)
```

easily?